### ISSUE TITLE
Null-values skipped in nested list

### ISSUE THREAD

**Issue by skywave** [bluelinelabs/LoganSquare#41]
I noticed an issue where in a (nested) list the null values are skipped, The list in the following json example has only 9 values instead of 10.
```java
@JsonObject
public class ApiCursor {
    @JsonField(name="Columns")
    private ArrayList<String> columns;

    @JsonField(name="Rows")
    private ArrayList<ArrayList<String>> rows;

    public ArrayList<String> getColumns() {
        return columns;
    }

    public void setColumns(ArrayList<String> columns) {
        this.columns = columns;
    }

    public ArrayList<ArrayList<String>> getRows() {
        return rows;
    }

    public void setRows(ArrayList<ArrayList<String>> rows) {
        this.rows = rows;
    }
}
```

```json
{
    "Rows": [
        [
            "value1",
            "value2",
            "value2",
            "value3",
            "value4",
            "value5",
            "value6",
            "value7",
            null,
            "value9",
            "value10"
        ]
    ],
    "Columns": [
        "column1",
        "column2",
        "column3",
        "column4",
        "column5",
        "column6",
        "column7",
        "column8",
        "column9",
        "column10",
        "column11"
    ]
}
```

**Comment by EricKuck** [comment]
This was actually done intentionally, but I'm not too sure if it was the right thing to do or not. Any idea what either Jackson or GSON do in this situation?

**Comment by skywave** [comment]
Gson puts a null in the list, don't know about the Jackson objectmapper, but i think it does.

**Comment by EricKuck** [comment]
Cool, thanks. I'll check out what Jackson does and, assuming it includes nulls, I'll make that change. If it doesn't, I'll try to come up with a good way to make it configurable.

**Comment by EricKuck** [comment]
This has now been pushed to master is available on JCenter as version 1.1.0.
